### DISCUSSION TITLE
Class "App\Models\BlindIndex" not found issue

### DISCUSSION THREAD

**Discussion by grafxflow** [spatie/laravel-ciphersweet#17] [Q&A] [answered]
I am getting the following error when trying to update the users emails.

```
php artisan ciphersweet:encrypt "App\Models\User" <CIPHERSWEET_KEY>
```

```
 Class "App\Models\BlindIndex" not found
```

The only way I have found to sort this is changing the code to the following.

```
public static function configureCipherSweet(EncryptedRow $encryptedRow): void
{
     $encryptedRow
     ->addField('email')
     ->addBlindIndex('email', new \ParagonIE\CipherSweet\BlindIndex('email_index'));
}
```

Is there something I am missing?

**Comment by acewebsa** [comment]
I've been trying this locally in laragon environment. I found that I needed to do as you pointed out below. After I did this, however I got the following exception:
Uncaught SodiumException: Not using the native libsodium bindings

What I found fixed this, was making sure that the sodium extension is enabled for php. Restart apache and it worked. For anyone that encounters the same issue.

**Comment by chuckienorton** [comment, marked as answer]
Adding `use ParagonIE\CipherSweet\BlindIndex;`to the top of the file worked for me.
